src/component/calculator.js

```diff
@@ -1,47 +1,57 @@
 import React from 'react';
 import './calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
+
+    this.handleClicks = this.handleClicks.bind(this);
   }
 
+  handleClicks = (e) => {
+    this.setState((state) => calculate(state, e.target.textContent));
+  };
+
   render() {
+    const { next, operation, total } = this.state;
     return (
       <div className="mainContainer">
         <div className="calculateDiv">
           <div>
-            <input type="text" className="inputResult" placeholder="0" disabled />
+            <div type="text" className="inputResult">
+              {next || operation || total || 0}
+            </div>
           </div>
           <div className="pressValue">
-            <p>AC</p>
-            <p>+/-</p>
-            <p>%</p>
-            <p className="orange">+</p>
+            <button type="button" onClick={this.handleClicks}>AC</button>
+            <button type="button" onClick={this.handleClicks}>+/-</button>
+            <button type="button" onClick={this.handleClicks}>%</button>
+            <button type="button" onClick={this.handleClicks} className="orange">+</button>
           </div>
           <div className="pressValue">
-            <p>7</p>
-            <p>8</p>
-            <p>9</p>
-            <p className="orange">X</p>
+            <button type="button" onClick={this.handleClicks}>7</button>
+            <button type="button" onClick={this.handleClicks}>8</button>
+            <button type="button" onClick={this.handleClicks}>9</button>
+            <button type="button" onClick={this.handleClicks} className="orange">X</button>
```

Comment:
- [ ] Great job here 👌💯, but I noticed that all operations are working well except the multiplication operation because you have to use a small `x` instead of a capital `X`, so kindly fix it 👌

https://user-images.githubusercontent.com/45231276/184264132-1d815355-cf54-4944-acde-c0b76e5f8ab0.mp4

Reply:
Hello @Bondok6 
Thank you